app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.java

```diff
@@ -142,6 +142,9 @@ private void fetchUserContributions() {
                     for (int count : editorTaskCounts.getDescriptionEditsPerLanguage().values()) {
                         totalEdits += count;
                     }
+                    for (int count : editorTaskCounts.getCaptionEditsPerLanguage().values()) {
+                        totalEdits += count;
```

Comment:
We will have an empty map if the user has no caption edits, which will result in the error when we try to access its values:
![error](https://user-images.githubusercontent.com/9540770/59232716-6c514280-8b9a-11e9-8505-0dfd4fb2ec07.png)

Reply:
Whoops, good catch. This should be fixed now.